### PR TITLE
Network fees infomation

### PR DESCRIPTION
**Pull request by AnthonyLaw**
Resolved: #267 

### Added
- Statistics Menu
- Recommended Network fees on Statistics page
- Effective Rental Fees on Statistics page
- getAverageNetworkFees

### Recommended Network fees
myself define the network rate for`Fast : 3` and `Slow : 0.5` 
taking the `rate * averageNetworkFees` = `Recommended Network fees`

### Effective Rental Fees
I refer the [calculation here](https://github.com/nemtech/catapult-rest/issues/98#issuecomment-562087709)

`namespaceRentalEffectiveFee = rootNamespaceRentalFeePerBlock * medianNetworkMultiplier`

medianNetworkMultiplier = 0.000282
rootNamespaceRentalFeePerBlock = 0.000001

The fees i will get ` 0.000000000282` with fix 6 decimal `0.000000` which is something wrong.

need input from you guys.
@dgarcia360  @evias

### CODE REVIEW

**Comment by dgarcia360** [comment]
Hi @AnthonyLaw, you might want to wait a bit until this is merged to just call one endpoint: https://github.com/nemtech/catapult-rest/pull/251

File: src/infrastructure/getStatistics.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  return averageNetworkFees
+}
+
+export const getNetworkTransactionFees = async () => {
+  const averageNetworkFees = await getAverageNetworkFees()
+  return Constants.NetworkConfig.NETWORK_FEES_RATE.map(feeRate => ({
+    netoworkFeesType: feeRate.TYPE,
+    fees: (feeRate.RATE * averageNetworkFees).toFixed(Constants.NetworkConfig.NATIVE_MOSAIC_DIVISIBILITY),
+    remark: 'XEM / Transfer'
```
Comment (dgarcia360) [inline src/infrastructure/getStatistics.js]: XYM?

File: src/infrastructure/getStatistics.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * limitations under the License.
+ *
+ */
+
+import sdkBlock from '../infrastructure/getBlock'
+import Constants from '../config/constants'
+
+const getAverageNetworkFees = async () => {
+  const maxDifficultyBlocks = await sdkBlock.getBlocksFromHeightWithLimit(Constants.NetworkConfig.MAX_DIFFICULTY_BLOCKS)
+  const maxDifficultyBlocksFeeMultipliers = maxDifficultyBlocks.map(({ feeMultiplier }) => parseFloat(feeMultiplier) > 0 ? parseFloat(feeMultiplier) : Constants.NetworkConfig.DEFAULT_DYNAMIC_FEE_MULTIPLIER / Math.pow(10, Constants.NetworkConfig.NATIVE_MOSAIC_DIVISIBILITY))
```
Comment (dgarcia360) [inline src/infrastructure/getStatistics.js]: Constants.NetworkConfig.DEFAULT_DYNAMIC_FEE_MULTIPLIER only applies to rental fees, not to transaction fees recommendation.

File: src/infrastructure/getStatistics.js (hunk 1 above)
Comment (AnthonyLaw) [reply src/infrastructure/getStatistics.js]: yes, XYM will change in the Rebranding branch.

File: src/infrastructure/getStatistics.js (hunk 2 above)
Comment (AnthonyLaw) [reply src/infrastructure/getStatistics.js]: right, thanks for clearify

**Comment by AnthonyLaw** [comment]
@dgarcia360 Glad to know that, Thanks.

**Comment by AnthonyLaw** [comment]
refer #388

**Pull request closed without merging**